lib/v8-to-istanbul.js

```diff
@@ -51,7 +52,28 @@ module.exports = class V8ToIstanbul {
     if (this.rawSourceMap) {
       if (this.rawSourceMap.sourcemap.sources.length > 1) {
         this.sourceMap = await new SourceMapConsumer(this.rawSourceMap.sourcemap)
-        this.covSources = this.sourceMap.sourcesContent.map((rawSource, i) => ({ source: new CovSource(rawSource, this.wrapperLength), path: this.sourceMap.sources[i] }))
+        // https://github.com/istanbuljs/v8-to-istanbul/issues/166
+        if (!this.sourceMap.sourcesContent) {
+          const fileList = this.sourceMap.sources.map(relativePath => {
+            const realPath = resolve(dirname(this.path), relativePath)
+            return new Promise(resolve =>
+              access(realPath, err => resolve(err ? null : readFile(realPath, 'utf-8')))
```

Comment:
? By [the docs](https://nodejs.org/api/fs.html#fsaccesspath-mode-callback),

> Do not use `fs.access()` to check for the accessibility of a file before calling `fs.open()`, `fs.readFile()` or `fs.writeFile()`. Doing so introduces a race condition, since other processes may change the file's state between the two calls. Instead, user code should open/read/write the file directly and handle the error raised if the file is not accessible.

Reply:
In the latest patch, I've used catch instead of access.